.github/workflows/ci.yml

```diff
@@ -63,7 +63,13 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: test
-        args: --all --features unstable attributes
+        args: --all --features "unstable attributes"
```

Comment:
The argument here was wrong.
The argument here was wrong.
"attributes" was not recognized as an argument to features.